doc/README.md

```diff
@@ -167,3 +167,12 @@ later steps are identical, so we've covered those steps in one place.
     -   Open `rsyslog/doc/build/index.html` in a browser.
     -   Use Calibre, Microsoft Edge, Okular, Google Play Books, or any other
         EPUB compatible reader to view the `rsyslog/doc/build/rsyslog.epub` file.
+
+---
+
+## CI deployment to GitHub Pages
+
+-   On pushes to the `main`/`master` branches and on manual runs, GitHub Actions builds the docs using the rsyslog doc Makefile: `make html SPHINXOPTS="-W"`.
```

Comment:
this is invalid and needs to be removed. doc build and artifact storage is just temporary per PR